### PR TITLE
Create acta-ichthyologica-et-piscatoria.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/76794/style-request-acta-ichthyologica-et-piscatoria

### CODE REVIEW

File: acta-ichthyologica-et-piscatoria.csl [hunk 1]
```diff
@@ -4,10 +4,11 @@
     <title>Acta Ichthyologica et Piscatoria</title>
     <id>http://www.zotero.org/styles/acta-ichthyologica-et-piscatoria</id>
     <link href="http://www.zotero.org/styles/acta-ichthyologica-et-piscatoria" rel="self"/>
-    <link rel="template" href="http://www.zotero.org/styles/wetlands"/>
+    <link href="http://www.zotero.org/styles/wetlands" rel="template"/>
     <link href="https://aiep.pl/main/instr.php" rel="documentation"/>
     <author>
-      <name>Sebastian Karcher</name>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
```
Comment (rmzelle) [inline acta-ichthyologica-et-piscatoria.csl]: @adam3smith, @POBrien333, I assume Patrick just forgot to update the name here.

File: acta-ichthyologica-et-piscatoria.csl [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      <key macro="author"/>
+      <key variable="author" sort="ascending"/>
+    </sort>
+    <layout>
+      <group delimiter=". ">
+        <group delimiter=". " suffix=".">
+          <text macro="author" font-weight="bold"/>
+          <text macro="year"/>
+          <text macro="title"/>
+          <choose>
```
Comment (adam3smith) [inline acta-ichthyologica-et-piscatoria.csl]: Just an FYI if you're looking at the previews -- citeproc-ruby doesn't apply group delimiters within `choose` elements contained in the group. This is fine with other citeprocs and I think should be changed in citeproc-ruby (which is doing this technically correctly according to the specs) and it's definitely fine in the styles, so just mentioning this so you don't worry about it.

Comment (rmzelle) [reply acta-ichthyologica-et-piscatoria.csl]: > This is fine with other citeprocs and I think should be changed in citeproc-ruby (which is doing this technically correctly according to the specs) and it's definitely fine in the styles, so just mentioning this so you don't worry about it.

My thinking about this has always been that style authors should be able to add `<choose/>` elements (e.g. to cover exceptions) without affecting the original functioning of the style, so yes, we should clarify this in the spec if needed and get citeproc-ruby to push the delimiter down into the `<choose/>` contents.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**